buildSrc/src/main/groovy/com/liferay/docker/workspace/environments/Config.groovy

```diff
@@ -191,6 +191,12 @@ class Config {
 			this.dockerContainerDatabase = "${this.namespace}-database-postgres"
 		}
 
+		if (this.services.contains("sqlserver")) {
```

Comment:
this applies to everywhere else -- i considered using `mssql` rather than `sqlserver`, since (1) it's shorter in length and (2) how the image is referenced; however, i also noticed that it only differs by one letter from `mysql`, which could be very confusing to the human eye, so i chose to leave it as `sqlserver`, just to be clear which database this is.